### PR TITLE
Removed debug output that doesn't work with faketensors

### PR DESCRIPTION
**Pull request by Chillee**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #86021
* #85998
* #85993
* __->__ #85992

### CODE REVIEW

**Review by ngimel — APPROVED** [review]
Are you sure this output is not informative and no one will ever need it?

File: functorch/_src/aot_autograd.py [hunk 1]
```diff
@@ -397,14 +397,6 @@ def fake_fn(primals, tangents):
         with track_graph_compiling("forward"):
             compiled_fw_func = aot_config.fw_compiler(fw_module, deduped_flat_args)
 
-        if config.debug_partitioner:
-            fw_outs = call_func_with_args(compiled_fw_func, deduped_flat_args)
-            activation_sizes = 0
-            for out in fw_outs[_num_outs:]:
-                if isinstance(out, torch.Tensor):
-                    activation_sizes += out.storage().nbytes()
```
Comment (eellison) [inline functorch/_src/aot_autograd.py:405]: `activation_sizes += out.storage().size() * out.element_size()` ? 

alternatively add `nbytes` to TypedStorage

**Review by eellison — COMMENTED** [review]
this should still work with fake tensors

**Comment by Chillee** [comment]
@ngimel yeah I was the only one using it, and I don't find it that informative lol.

Basically, we already compute the "theoretical" activations saved after the min-cut, but I was curious if that diverged from the actual activations due to thing like expanded tensors or something.

But the difference has pretty much always been negligible in practice.

**Pull request closed without merging**